src/main/java/au/org/emii/netcdf/iterator/reader/NetcdfReader.java

```diff
@@ -76,4 +80,13 @@ public String getString(Set<IndexValue> indexValues) {
         }
     }
 
+    public String getDisplayValue(Set<IndexValue> indexValues) {
```

Comment:
That should be addressed if possible under https://github.com/aodn/netcdf-iterator/issues/2